packages/wp-now/src/tests/execute-php.spec.ts

```diff
@@ -12,7 +12,10 @@ test('php file execution in index mode', async () => {
 	const options = await getWpNowConfig({
 		path: exampleDir,
 	});
-	await executePHPFile(path.join(exampleDir, 'hello-world.php'), options);
+	await executePHP(
+		['php', path.join(exampleDir, 'hello-world.php')],
```

Comment:
For the purposes of having something closer to end to end tests, is it possible to include some `yargs.parse()` to parse arguments as the user might pass them, and then send those arguments to `executePHP()`?

Reply:
Done https://github.com/WordPress/wordpress-playground/commit/405a2a026c58ad989acc2a7622a5c4c6815affca !

Reply:
good call